commands/jail.js

```diff
@@ -2,39 +2,41 @@ const { SlashCommandBuilder, ButtonBuilder, ButtonStyle, ActionRowBuilder, Embed
 const rulesKimoState = require('../patterns/rulesKimoState');
 const Jail = require('../models/jailTracker');
 const jail = require('../patterns/jail');
+const { jailedRoleID, adminRoleID } = require('../ids.json');
 
 module.exports = {
     data: new SlashCommandBuilder()
-    .setName('jail')
-    .setDescription('jail someone.')
-    .addUserOption(option =>
-        option
-            .setName('target')
-            .setDescription('to jail')
-            .setRequired(true))
-    .addIntegerOption(option =>
-        option
-            .setName('time')
-            .setDescription('minutes to jail')
-            .setRequired(true))
-    .addStringOption(option =>
-        option
-            .setName('reason')
-            .setDescription('reason for jailing')
-            .setRequired(true)),
+        .setName('jail')
+        .setDescription('jail someone.')
+        .addUserOption(option =>
+            option
+                .setName('target')
+                .setDescription('to jail')
+                .setRequired(true))
+        .addIntegerOption(option =>
+            option
+                .setName('time')
+                .setDescription('minutes to jail')
+                .setRequired(true))
+        .addStringOption(option =>
+            option
+                .setName('reason')
+                .setDescription('reason for jailing')
+                .setRequired(true)),
 
     async execute(interaction, client) {
 
-        if (!interaction.member.roles.cache.get('1203377553763475497')) return interaction.reply({ content: `you cannot use this...`, ephemeral: true });
+        if (!interaction.member.roles.cache.get(adminRoleID)) 
+            return interaction.reply({ content: `you cannot use this...`, ephemeral: true });
 
         const jailTarget = await interaction.options.getMember('target');
 
-        if (jailTarget.roles.cache.get('1202749571957006348')) return interaction.reply({ content: `You cannot jail someone already in Jail!`, ephemeral: true });
+        if (jailTarget.roles.cache.get(jailedRoleID)) 
```

Comment:
it's kind of tricky to know which role/channel/server id string corresponds to what, and if any of these change in the future for some unforeseen reason, it'll really suck for you to have to replace it in each file. Importing from ids.json means you can keep track of all of them in one place, instead of having them copy/pasted everywhere.